BUILD

```diff
@@ -1111,3 +1178,74 @@ sh_test(
         "update_file_lists.sh",
     ],
 )
+
+java_proto_library(
```

Comment:
Visibility private on these, unless we mean for this BUILD file to be their ultimate location?

Reply:
Done! also fixed some inconsistent formatting.